app/views/service_requests/service_request_assignments/_add_card.html.slim

```diff
@@ -0,0 +1,49 @@
+- if !@account.have_payment_method
+  .payment_methods
+    div
+      = link_to 'Default Payment Method', '#', class: 'btn btn-default w100-pct text-center mt10'
+    div
+      = link_to 'Add New Card', service_requests_service_request_assignment_start_accepting_path(assignment.id, service_request_id: service_request.id), remote: true, class: 'btn btn-default w100-pct text-center mt10'
+    div
+      = link_to 'Done', '#', remote: true, class: 'btn btn-danger w100-pct text-center mt20'
+- else
+  .panel-body
+    .text-center
+      a.visa href='javascript:void(0);'
```

Comment:
Better to have `href="#"` here, add some class for that links you want not working and add event with jQuery that will do `e.preventDefault();`.